src/graphql/subscriptions/online-users.ts

```diff
@@ -0,0 +1,34 @@
+import { getEndOfWeek, getStartOfWeek } from '../../lib/utils';
+
+import { useMemo } from 'react';
+import { useSubscriptionWithCache } from '../../hooks';
+
+const ONLINE_USER_SUBSCRIPTION = `subscription {
+  user_online (
+    order_by: {
+      name: asc
```

Comment:
I wonder if it makes more sense to order by the timestamp or something to be able to view people as they join the session?

Like new joiners getting added to the top/bottom of the list?
Or perhaps we can put a "new" badge next to their name in the list?

Reply:
Yes but would require some changes, as the current solution relies on the timestamp being updated every three seconds. It would result in the list being reordered every three seconds, which would not be nice.